presto-parser/src/main/java/io/prestosql/sql/parser/AstBuilder.java

```diff
@@ -356,6 +357,13 @@ public Node visitRenameTable(SqlBaseParser.RenameTableContext context)
         return new RenameTable(getLocation(context), getQualifiedName(context.from), getQualifiedName(context.to));
     }
 
+    @Override
+    public Node visitCommentTable(SqlBaseParser.CommentTableContext context)
+    {
+        String comment = context.string() != null ? ((StringLiteral) visit(context.string())).getValue() : null;
```

Comment:
I find this form easier to read:

```java
Optional<String> comment = Optional.empy();

if (context.string() != null) {
    comment = Optional.of(((StringLiteral) visit(context.string())).getValue())
}

return new Comment(...)
```

See how other visitXXX methods are implemented.